niche_vlaanderen/codetables.py

```diff
@@ -23,7 +45,11 @@ def check_lower_upper_boundaries(df, min_col, max_col, value):
     """
 
     group_cols = set(df.columns.tolist()) - {min_col, max_col, value}
-    for sel_group, subtable in df.groupby(list(group_cols)):
+    if len(list(group_cols)) > 1:
```

Comment:
I suppose this is for pandas compatibility. Maybe add a comment.

Reply:
Good idea, I added inline comment